psi4/src/psi4/libmints/twobody.cc

```diff
@@ -77,7 +77,7 @@ TwoBodyAOInt::TwoBodyAOInt(const IntegralFactory *intsfactory, int deriv)
     else if (screentype == "CSAM")
         screening_type_ = ScreeningType::CSAM;
     else if (screentype == "DENSITY")
```

Comment:
Would it make sense to diable setting `screening density` altogether?

Reply:
It would be, yes. I have this planned in the future, although it could quite possibly be done in this PR as well.

Reply:
The reason this is still in here is to account for cases where density screening is used in the HF step and another form of screening is used through TwoBodyAOInt for a later step in the calculation. Looking through the code, it seems that an example of this is in the DirectJKGrad class, which uses TwoBodyAOInt screening functions in its tests for shell block significance.

This will certainly be removed once density screening is separated out as a keyword, though.